client/app/telecom/pack/xdsl/modem/router/dhcp/pack-xdsl-modem-dhcp.factory.js

```diff
@@ -46,7 +46,29 @@ angular.module('managerApp').factory('PackXdslModemDhcpObject', (OvhApiXdsl, $tr
         TucToast.success($translate.instant('xdsl_modem_dhcp_success', { name: self.domainName }));
         return data;
       }).catch((err) => {
-        TucToast.error($translate.instant('xdsl_modem_dhcp_submit_error'));
+        if (err.data && err.data.message) {
+          const regex = /\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b/g;
+          let ipsList = err.data.message.match(regex);
+          const ipUsed = ipsList[0];
+          ipsList = ipsList.filter(item => item !== ipUsed);
+          $uibModal.open({
+            animation: true,
+            templateUrl: 'app/telecom/pack/xdsl/modem/router/dhcp/pack-xdsl-modem-dhcp-modal.html',
+            controller: 'telecomXdslModemDhcpModal',
+            controllerAs: '$ctrl',
+            resolve: {
+              data: () => {
```

Comment:
All variables contain data, the name should convey what kind of data the variable holds :)